src/applications/disability-benefits/all-claims/tests/all-claims.cypress.spec.js

```diff
@@ -0,0 +1,107 @@
+import path from 'path';
+
+import testForm from 'platform/testing/e2e/cypress/support/form-tester';
+import { createTestConfig } from 'platform/testing/e2e/cypress/support/form-tester/utilities';
+
+import formConfig from '../config/form';
+import manifest from '../manifest.json';
+import { mockItf } from './all-claims.cypress.helpers';
```

Comment:
This is separate from the other mock responses that are loaded as fixtures because the Cypress fixtures functionality doesn't evaluate JS code when reading fixtures.

Fixtures in general aren't absolutely necessary, since you could import the modules like anything else, but they provide a shortcut for accessing static data that you use in your tests.